src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String COMMA = ",";
+    public static final String BUY = "buy";
+    public static final String SUPPLY = "supply";
+    public static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        supply = readAndCalculateFile(supply, buy, fromFileName)[0];
+        buy = readAndCalculateFile(supply, buy, fromFileName)[1];
```

Comment:
it is better to create a variable for readAndCalculateFile(supply, buy, fromFileName), because in this case you call the method twice readAndCalculateFile